build.gradle

```diff
@@ -3248,7 +3248,6 @@ project(':connect:runtime') {
     testImplementation project(':group-coordinator')
     
     testImplementation libs.junitJupiterApi
-    testImplementation libs.junitVintageEngine
```

Comment:
Please cleanup https://github.com/apache/kafka/blob/trunk/build.gradle#L447

Reply:
There is a little problem, If I remove the `testImplementation libs.junitVintageEngine`, Gradle can't get `org.junit.` dependencies and some annotations are error. I find a way to resolve it

Reply:
we should remove all usage of junit 4 code form connect module. If you get such error, please rewrite them by junit 5